mini_computer_api/resources/config.yaml

```diff
@@ -0,0 +1,5 @@
+# abbreviation of state in which the acquisition is happening
+state: 'NC'
+
+# first 2 letters of default image names which are generated on camera
+cam_img_name: 'NC'
```

Comment:
why have two variables with the same value? will these ever be different?

Reply:
They are different when people don't update the default names of images being taken on the camera, then the image names start from 'DSC' or something. I thought it would be better to just have it as 2 different variables.

Reply:
but you're only still using state for naming the image directory. is the two things were different it would break things with how you currently have it right?

Reply:
No it would not break anything, the second variable is to only facilitate finding images that come from camera in the directory. It's literally the same as before the changes, the only thing that changed is I made these variables.

Reply:
ok. can you remove the config file from this PR and add an example_config.yaml so people can see what should be there?

Reply:
what is the difference between both files?